integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -739,6 +756,10 @@ void testHiveTableCreatedByGravitino() throws InterruptedException {
     Assertions.assertTrue(
         data.contains("input_format = 'org.apache.hadoop.hive.ql.io.orc.OrcInputFormat'"));
     Assertions.assertTrue(data.contains("serde_lib = 'org.apache.hadoop.hive.ql.io.orc.OrcSerde'"));
+    Assertions.assertTrue(data.contains("bucket_count = 4"));
+    Assertions.assertTrue(data.contains("bucketed_by = ARRAY['booleantype']"));
+    Assertions.assertTrue(data.contains("partitioned_by = ARRAY['binarytype']"));
+    Assertions.assertTrue(data.contains("sorted_by = ARRAY['longtype']"));
```

Comment:
I think you'd better add more test cases here, currently one is not enough to cover all the senarios.

Reply:
I added more tests and tested the following case.
- Create partitioning, bucketing, and sorting by table by Trino and load it using Gravitino.
- Create partitioning, bucketing, and sorting by table by Gravitino and load it using Trino.